ProjectWork/Pages/Artist/DetailArtist.razor

```diff
@@ -0,0 +1,133 @@
+﻿@page "/detail-artist/{Id:int}"
+@using ProjectWork.Models.Artwork
+@using ProjectWork.ViewModels.Artist
+@using ProjectWork.Models.Artist
+@inject SearchArtistVM ArtistDirector
+@inject IJSRuntime JsRuntime
+
+<div id="page-content">
+
+    <!-- ======= End Page Header ======= -->
+    <div class="page-header d-flex align-items-center">
+        <div class="container position-relative">
+            <div class="row d-flex justify-content-center">
+                <div class="col-lg-6 text-center">
+                    <h2>@_artist.Name</h2>
+                </div>
+            </div>
+        </div>
+    </div>
+    <!-- End Page Header -->
+
+
+    <section id="gallery-single" class="gallery-single">
+        <div class="container">
+
+            <div class="position-relative h-100">
+                <div class="slides-1 portfolio-details-slider swiper">
+                    <div class="swiper-wrapper align-items-center">
+
+                        <div class="swiper-slide">
+                            <a href="@_artist.Portrait" data-fancybox="gallery" data-caption="@_artist.Name">
+                                <img class="w3-round w3-image w3-margin-bottom" src="@_artist.Portrait" alt="@_artist.Name" />
```

Comment:
Since the artistis portraits are very low the image resoultion there should be a different HTML compared to Artwork...